databricks/koalas/base.py

```diff
@@ -520,3 +520,66 @@ def any(self, axis: Union[int, str] = 0) -> bool:
             return False
         else:
             return ret
+
+    # TODO: add frep and axis parameter
+    def shift(self, periods=1, fill_value=None):
+        """
+        Shift Series/Index by desired number of periods.
+
+        .. note:: the current implementation of shift uses Spark's Window without
+            specifying partition specification. This leads to move all data into
+            single partition in single machine and could cause serious
+            performance degradation. Avoid this method against very large dataset.
+
+        Parameters
+        ----------
+        periods : int
+            Number of periods to shift. Can be positive or negative.
+        fill_value : object, optional
+            The scalar value to use for newly introduced missing values.
+            The default depends on the dtype of self. For numeric data, np.nan is used.
+
+        Returns
+        -------
+        Copy of input Series/Index, shifted.
+
+        Examples
+        --------
+        >>> df = ks.DataFrame({'Col1': [10, 20, 15, 30, 45],
+        ...                    'Col2': [13, 23, 18, 33, 48],
+        ...                    'Col3': [17, 27, 22, 37, 52]})
```

Comment:
This equivalent example in `DataFrame` needs `columns=[...]`. Could you fix it too?

Reply:
Could you also add it to https://github.com/databricks/koalas/blob/a248cc6788e294aebf03d9344429872512cd1db4/databricks/koalas/frame.py#L1901-L1904 as well?

Reply:
Yup!